pom.xml

```diff
@@ -26,6 +26,8 @@
     <java.level>6</java.level>
     <!-- Use the same configuration than before. -->
     <concurrency>2</concurrency>
+    <permgen.size>128m</permgen.size>
```

Comment:
I don't think this is needed based on the current 160 MB setting already in the master branch.